src/Particle/ParticleSet.cpp

```diff
@@ -826,6 +806,15 @@ void ParticleSet::loadWalker(Walker_t& awalker, bool pbyp)
   activePtcl = -1;
 }
 
+void ParticleSet::mw_loadWalker(const RefVectorWithLeader<ParticleSet>& psets,
+                                const RefVector<Walker_t>& walkers,
+                                bool pbyp)
+{
+#pragma omp parallel for
```

Comment:
The strange looking nested OMP is already called from a crowd concurrent section so I wonder why it needs more threads for what looks like a copy from one object to another. Then I reread load walker. 

I'm not sure the driver should know about the distance tables but an important cost of the algorithm is hidden from view here. The method documentation should perhaps emphasize that the this is actually a hot spot in some cases.

Reply:
loadWalker can be called outside the crowd scope now in WalkerControl. It can be a hot spot of quite a few walkers got exchanged and need to recompute full distance table.